src/jvm/main/org/jetbrains/lincheck/Lincheck.kt

```diff
@@ -27,27 +32,30 @@ object Lincheck {
      *
      * @param invocations number of different interleavings of code in the [block] that should be explored.
      * @param block lambda which body will be a target for the interleavings exploration.
+     *
+     * @throws LincheckAssertionError in case if some exception was discovered.
```

Comment:
please check the grammar. Please also check the rest of the GPMC API documentation.